### PR TITLE
Add prettier to auto-kol/agent

### PR DESCRIPTION
**Pull request by jfrank-summit**
In order to standardize the code formatting I have added prettier. @marc-aurele-besner is more experienced and probably opinionated regarding formatting, so please review and feel free to improve.

I will also be adding linting after we get through some refactoring.

### CODE REVIEW

File: auto-kol/agent/.prettierrc.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export default {
+  semi: true,
```
Comment (marc-aurele-besner) [inline auto-kol/agent/.prettierrc.js:2]: I see you like your semicolon 😂

**Review by marc-aurele-besner — APPROVED** [review]
All good with me, I'll just suggest also adding a .vscode/settings.json with similar lines as [here](https://github.com/autonomys/astral/blob/683c489621030b6855e11354be745d40120aacd6/.vscode/settings.json#L2-L7) to avoid actually having to remember to run prettier each time we are committing something

File: auto-kol/agent/.prettierrc.js (hunk 1 above)
Comment (jfrank-summit) [reply auto-kol/agent/.prettierrc.js:2]: I'm indifferent to semi-colon, it just seems more popular 🙂. I can remove it if it is going to drive you crazy!

**Comment by jfrank-summit** [comment]
> All good with me, I'll just suggest also adding a .vscode/settings.json with similar lines as [here](https://github.com/autonomys/astral/blob/683c489621030b6855e11354be745d40120aacd6/.vscode/settings.json#L2-L7) to avoid actually having to remember to run prettier each time we are committing something

I have a local vscode settings that runs on save. I know you like committing those to the repo. Another one I'm pretty indifferent on.

File: auto-kol/agent/.prettierrc.js (hunk 1 above)
Comment (marc-aurele-besner) [reply auto-kol/agent/.prettierrc.js:2]: haha no stress

**Review by Xm0onh — APPROVED** [review]

**Pull request merged**